2024/MODULE_1.md

```diff
@@ -0,0 +1,331 @@
+
+- [Pedersen Commitments](https://asecuritysite.com/encryption/ped)
+- [What is a Pedersen Commitment?](https://crypto.stackexchange.com/questions/64437/what-is-a-pedersen-commitment)
+
+As a final touch on this section, [this video [10:32]](https://www.youtube.com/watch?v=IkNZWJFcfcU) provides a brief yet comprehensive summary of the fundamental concepts discussed in this section.
+
+:::info
+**🤔 Consider the following:**
+1. What is the main purpose of Cryptographic Commitments?
+2. What type of information does a Polynomial Commitment hide and reveal?
+3. How do Pedersen Commitments contribute to privacy in cryptography?
+4. Why are Cryptographic Commitments important in blockchain technology?
+:::
+
+### Intro to Elliptic Curve Cryptography
+
+Elliptic curve cryptography builds upon traditional cryptographic principles by applying them to the mathematical structure of elliptic curves.
+
+For example, there exists an **Elliptic Curve Discrete Log Problem (ECDLP)**. By using elliptic curves, it has certain benefits over the regular Discrete Log Problem (DLP). These links are a good place to start:
+
+- [Elliptic Curves - Computerphile [8:41]](https://www.youtube.com/watch?v=NF1pwjL9-DE)
+- [A (relatively easy to understand) primer on elliptic curve cryptography](https://blog.cloudflare.com/a-relatively-easy-to-understand-primer-on-elliptic-curve-cryptography/)
+
+**Elliptic Curve Digital Signature Algorithm (ECDSA)** extends the traditional Digital Signature Algorithm (DSA) by incorporating elliptic curves as well, enabling stronger security with shorter key lengths. This makes the algorithm more efficient and suitable for constrained environments. ECDSA is widely used in various secure communication protocols, including SSL/TLS.
+
+The following StackExchange answer goes over the differences between some of the most popular digital signature protocols. It also leads into our next topic, which is another type of digital signature called the Edwards-curve Digital Signature Algorithm (EdDSA).
+
+- [What is the difference between the RSA, DSA, and ECDSA keys that ssh uses?](https://askubuntu.com/a/1000928/733503)
+
+### BLS Signatures
+
+TODO - keep this simple and black-boxed
+
+BLS (Boneh-Lynn-Shacham) signatures are a type of cryptographic signature scheme that allows for efficient aggregation of individual signatures into a single signature. It also makes use of pairing-based cryptography.
+
+The intuition of BLS is that it is an aggregation signature. The image below shows how the public key and signature are aggregated. The advantage is that all signatures can be verified at once. That's why it is commonly used in consensus protocols.
+![BLS](./assets/bls-signatures.jpeg)
+
+### Pairing-Based Cryptography
+
+TODO - keep this simple and black-boxed
```

Comment:
Is this done?

Reply:
@NOOMA-42 Do you feel like this is simple and black-boxed enough?